src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/ParseJson_Coercions.txt

```diff
@@ -0,0 +1,307 @@
+﻿// The If function will attempt to coerce the else clause to the same type as the then clause
+// this allows us to control the exact expected coercion type
+>> If(false, 4, ParseJSON("5"))
```

Comment:
can you add:  (this helps with documentation)

>> If(false, "4", ParseJSON("5"))
"4"

Reply:
Done

Reply:
I mean where the true-arg is a string (vs. where true-arg is a number), but all else is the same. It's showing coercion to true-args.